mappings/net/minecraft/entity/boss/dragon/EnderDragonEntity.mapping

```diff
@@ -17,9 +17,15 @@ CLASS ane net/minecraft/entity/boss/dragon/EnderDragonEntity
 	METHOD U kill ()V
 	METHOD a damage (Laft;F)Z
 		ARG 1 source
+		ARG 2 amount
 	METHOD a damage (Lanc;Laft;F)Z
 		ARG 1 part
 		ARG 2 source
+		ARG 3 amount
+	METHOD a crystalWasDamaged (Land;Let;Laft;)V
```

Comment:
possibly drop the was? `crystalDamaged? `

Reply:
Should be `crystalDestroyed` or similar.